tobac/utils/bulk_statistics.py

```diff
@@ -266,10 +266,16 @@ def get_statistics_from_mask(
          Updated feature dataframe with bulk statistics for each feature saved in a new column
     """
     # warning when feature labels are not unique in dataframe
-    if not features.feature.is_unique:
-        raise logging.warning(
+    if not features[id_column].is_unique:
+        logging.warning(
             "Feature labels are not unique which may cause unexpected results for the computation of bulk statistics."
         )
+    # extra warning when feature labels are not unique in timestep
+    uniques = features.groupby("time")[id_column].value_counts().values
```

Comment:
I think by "time" would make more sense for this warning because we actually use the time dimension to perform the bulk statistics for each unique feature on: 

https://github.com/tobac-project/tobac/blob/698d53bf019de6542056272c01d6a472caaf2c30/tobac/utils/bulk_statistics.py#L288-L298

Reply:
But you have a good point. The latter could done by "frame", too. So lets just briefly discuss this in the next dev meeting maybe!

Reply:
Personally I prefer using `"time"` here, but can see also why `"frame"` would also be appropriate (e.g. in edge cases like Sean mentioned where dataframe times are saved to ms accuracy rather than ns causing mismatches. Maybe we can leave it as is and revisit if it causes any issues in future